<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>499</width>
    <height>662</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Yrsa Light</family>
    <pointsize>14</pointsize>
    <italic>false</italic>
    <bold>true</bold>
    <underline>false</underline>
    <strikeout>false</strikeout>
    <kerning>true</kerning>
   </font>
  </property>
  <property name="windowTitle">
   <string>Weather app</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>assets/appicon.png</normaloff>assets/appicon.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="inputMethodHints">
   <set>Qt::ImhNone</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_search">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>580</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
    <property name="icon">
     <iconset theme="system-search"/>
    </property>
   </widget>
   <widget class="QLabel" name="temp_today">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>180</y>
      <width>72</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>today_temp</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="search_box">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>530</y>
      <width>331</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="city">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>72</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Mashhad</string>
    </property>
   </widget>
   <widget class="QLabel" name="icon_location">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="assets/ui_resources.qrc">:/location icon/location_icon/icons8-location-64(2).png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="icon_now">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>120</y>
      <width>72</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>icon_now</string>
    </property>
   </widget>
   <widget class="QLabel" name="text1">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>30</y>
      <width>51</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Today</string>
    </property>
   </widget>
   <widget class="QLabel" name="text2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>280</y>
      <width>51</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Wind:</string>
    </property>
   </widget>
   <widget class="QLabel" name="text3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>280</y>
      <width>51</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>km/h</string>
    </property>
   </widget>
   <widget class="QLabel" name="background_label">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>-20</y>
      <width>511</width>
      <height>661</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/newPrefix/background_images/wind-spring-white-stratosphere-pure.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="assets/ui_resources.qrc">:/newPrefix/background_images/wind-spring-white-stratosphere-pure.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>350</y>
      <width>451</width>
      <height>161</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QLabel" name="text5">
       <property name="text">
        <string>Mon</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="icon_tomo">
       <property name="styleSheet">
        <string notr="true">image: url(:/newPrefix/icons/dark/png/00.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="text4">
       <property name="text">
        <string>Tomorrow</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="text6">
       <property name="text">
        <string>Tue</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="icon_ntomo">
       <property name="text">
        <string>icon_now</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="icon_nntomo">
       <property name="text">
        <string>icon_now</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>background_label</zorder>
   <zorder>btn_search</zorder>
   <zorder>temp_today</zorder>
   <zorder>search_box</zorder>
   <zorder>city</zorder>
   <zorder>icon_location</zorder>
   <zorder>icon_now</zorder>
   <zorder>text1</zorder>
   <zorder>text2</zorder>
   <zorder>text3</zorder>
   <zorder>frame</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>499</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="assets/ui_resources.qrc"/>
  <include location="../../../../../../home/tolo/.designer/backup/assets/ui_resources.qrc"/>
 </resources>
 <connections/>
</ui>
